microsoftgraph/microsoft-graph-explorer-v4
# Fix: Add method to filter ResizeObserver exceptions

## Overview

Closes #2303 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Checkout to this branch.
* Run local instance of GE.
* Navigate to the first track request under the Network tab.
* Observe absence of telemetry captured in the image below

![image](https://user-images.githubusercontent.com/34104277/212099256-ea40cb5e-d8aa-48ca-9db6-0bd938388421.png)
